src/main/java/rotateit/service/github/label/OnLabelEventToViewListener.java

```diff
@@ -0,0 +1,14 @@
+package rotateit.service.github.label;
+
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import rotateit.domain.Event;
+import rotateit.domain.github.label.LabelPayload;
+
+@Component
```

Comment:
you already defined it as a bean in `ApplicationContext` no need for `@Component`, we are not using component scanning